src/package_io/parse_input.star

```diff
@@ -296,7 +292,7 @@ def parse_network_params(input_args):
     ):
         fail("electra can only happen with capella genesis not bellatrix")
 
-    required_num_validators = 2 * result["network_params"]["slots_per_epoch"]
```

Comment:
why are you hard coding this?

Reply:
fixed

Reply:
oh my multi tasking is failing, i didn't see that this depends on `slots_per_epoch`